tools/repeatexplorer2/repex_full_clustering.xml

```diff
@@ -0,0 +1,259 @@
+      &&
+
+      export PYTHONHASHSEED=0
+      &&
+
+      ## output will go here
+      mkdir -p '${ReportFile.extra_files_path}'
+      &&
+
+      /repex_tarean/seqclust
+      --cpu \${GALAXY_SLOTS:-1}
+      --max_memory \${GALAXY_MEMORY_KB}
+      '${paired}'
+      #if $sample:
+        --sample '${sample}'
+      #end if
+      --taxon '${taxon}'
+      --output_dir='${ReportFile.extra_files_path}'
+      #if $advanced.mincl:
+        --mincl '${advanced.mincl}'
+      #end if
+      --assembly_min '${advanced.assembly_min}'
+      #if $advanced.keep_names:
+        --keep_names
+      #end if
+      '${FastaFile}'
+      &&
+
+      ## archive the output
+      tar -cvf '${ReportArchive}' --directory='${ReportFile.extra_files_path}' .
+      &&
+
+      ## pick up the html index
+      cp '${ReportFile.extra_files_path}/index.html' ./index.html
+
+      ]]></command>
+  <inputs>
+    <param name="FastaFile" label="NGS reads" type="data" format="fasta" help="Input file must contain FASTA-formatted NGS reads. Illumina paired-end reads are recommended."/>
+    <param argument="--paired" type="boolean" truevalue="--paired" falsevalue="" checked="True" label="Paired-end reads" help="If paired-end reads are used, they must be interleaved and all pairs must be complete. Example of the correct format is provided in the help below."/>
+    <param argument="--sample" type="integer" optional="true" label="Subsample reads (number)" help="Use an integer &gt; 1 to select a specific number of reads to use. Leave this field blank to use the entire dataset."/>
+    <param argument="--taxon" label="Select taxon and protein domain database version (REXdb)" type="select" help="Reference database of transposable element protein domains - REXdb - is used for annotation of repeats">
+      <option value="VIRIDIPLANTAE3.0" selected="true">Viridiplantae version 3.0 </option>
+      <option value="VIRIDIPLANTAE2.2" selected="true">Viridiplantae version 2.2</option>
+      <option value="METAZOA3.0">Metazoa version 3.0</option>
+      <option value="METAZOA2.0">Metazoa version 2.0</option>
+    </param>
+    <section name="advanced" title="Advanced options" expanded="false">
+      <param argument="--mincl" label="Cluster size threshold  for detailed analysis" type="float" value="" min="0.0001" max="100" optional="true" help="Minimal size (as percentage of input reads) of the smallest cluster which is analyzed; clusters with less than 20 reads are not considered."/>
+      <param argument="--assembly_min" type="integer" label="Minimal cluster size for assembly" value="5" min="2" max="100"/>
+      <param argument="--keep_names" label="Keep original read names" type="boolean" checked="false" help="By default, reads are renamed using integers. Use this option to keep original names."/>
+    </section>
+  </inputs>
+  <outputs>
+    <data name="ReportArchive" format="tar" label="RepeatExplorer2 - Archive with HTML report from data ${FastaFile.hid}"/>
```

Comment:
can we make this tar optional? and disabled by default?

```suggestion
    <data name="ReportArchive" format="tar" label="RepeatExplorer2 - Archive with HTML report from on ${on_string}"/>
```

Reply:
You're right, we don't need it, it's all accessible from the html page and it's better to add the individual outputs if we want them in Galaxy.

Removed it.